kolibri/plugins/coach/assets/src/vue/groups-page/move-learners-modal.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+
+  <core-modal :title="$tr('moveLearners')"
+    @cancel="close">
```

Comment:
>> If you are referring to @cancel being on the next line, that is intentional.

> All or none, bro

Yeah, "all or none" was my thinking, too.

That said, we're not at all consistent about multi-line tags/attributes (indentation and location of closing `>`), so we can punt on making a rule or a change here for now. I want to start a conversation about that and potentially add a lint rule after this feature push.